app/config.py

```diff
@@ -8,6 +8,11 @@ class Settings(BaseSettings):
     API_V2_STR: str = "/api/v2"
     admin_email: str = "devnull@ncsa.illinois.edu"
 
+    # openssl rand -hex 32
+    local_auth_secret = (
+        "47358d6ace318031e822d722c15d191ba0d3e2cc7594317514e553424ccf3e39"
```

Comment:
is it safe to commit this in to the repo?

Reply:
It's randomly generated with the command above `openssl rand -hex 32`. We could leave it blank but a developer would have to set it up first. On the other hand leaving it blank means no accidental reuse of this specific one. 

This will probably go away with keycloak.